pillar.example

```diff
@@ -335,6 +335,12 @@ openssh:
     static:
       github.com: 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGm[...]'
       gitlab.com: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsj2bN[...]'
+    # Prevent an ever-changing ssh_known_hosts file caused by a domain which
+    # is served from multiple IP addresses.
+    omit_ip_address:
+      - github.com
+    # Disable completely:
+    #omit_ip_address: true
```

Comment:
This is failing the `yamllint` on two counts:

* https://travis-ci.com/saltstack-formulas/openssh-formula/jobs/277659800#L298-L303

This is the easiest way to get it to work, rather than adding `yamllint` disabling comments:

```yaml
    # Prevent an ever-changing ssh_known_hosts file caused by a domain which
    # is served from multiple IP addresses.
    # To disable completely:
    # omit_ip_address: true
    # Or to disable by specific hosts:
    omit_ip_address:
      - github.com
```

Reply:
Thanks @myii, you're fast as ever. :-) Thanks for helping me with `yamllint`!
I will implement the changes.

Reply:
Done.